tasks/random_dot_motion.md

```diff
@@ -0,0 +1,43 @@
+# Summary
+> Participants are asked to answer which direction (0 to 360 degrees) a circular field of 50 dots are moving.
+
+# References
+> Dynamics of judgment propagation. Mehdi Moussaïd, Stefan M. Herzog, Juliane E. Kämmer, Ralph Hertwig. Proceedings of the National Academy of Sciences Apr 2017, 114 (16) 4117-4122; DOI: 10.1073/pnas.1611998114
+
+# Stimuli
+## The visual components
+> Video of dot movement: http://movie-usa.glencoesoftware.com/video/10.1073/pnas.1611998114/video-1
+
+## Materials for alternative versions of the experiment 
+> There are 2 versions of the study. In version 1,  2 individuals were paired. In study 2, 6 participants were paired in a chain.
+
+# Procedure
+## Steps
+> There are three difficulty levels. The three difficulty levels were characterized by
+different proportions of correlated dots, pc = 50%, 30%, and 5%. In addition, the deviation was set to 0° in difficulty level 1 and to 45° in
+difficulty levels 2 and 3.
+
+ > There are 2 versions of the study
+ > Version 1: Individuals were paired with the same partner for 15 consecutive rounds. In each round, the individual made a guess, then looked at their partner's estimate, and finally revised their guess. Partners always had the same correct angle programmed into the dots, but the difficulty level could vary by trial between the pair. 
+ > Version 2: A group of 6 participants (A, B...F) are connected along a chain. After each initial guess, A's estimate is shown to B, B's estimate is shown to C, ... E's estimate is shown to F. This was repeated across 15 rounds. A (head of chain) always experienced difficultu level 1, and all other participants in the chain experinced difficulty level 3. 
```

Comment:
Similar comment to the one above, should this be broken out into steps: 
1. Six people are given specific letters (A, B, C, D, E, F) in a chain 
2. Each person in the chain is paired with their predecessor for 15 consecutive rounds and the level of difficulty increases further down the chain (A-head of the chain always has difficulty level one and difficulty increases along the chain)
3. The individual at position k observed the final estimate of the individual at position k ‒ 1 before entering their final estimate